packages/core/http/core-http-router-server-internal/src/router.ts

```diff
@@ -143,13 +148,47 @@ export class Router<Context extends RequestHandlerContextBase = RequestHandlerCo
         const routeSchemas = routeSchemasFromRouteConfig(route, method);
 
         this.routes.push({
-          handler: async (req, responseToolkit) =>
-            await this.handle({
+          handler: async (req, responseToolkit) => {
+            const startUtilization = performance.eventLoopUtilization();
+            const start = performance.now();
+
+            const response = await this.handle({
               routeSchemas,
               request: req,
               responseToolkit,
               handler: this.enhanceWithContext(handler),
-            }),
+            });
+
+            const { active, utilization } = performance.eventLoopUtilization(startUtilization);
+
```

Comment:
I think we could fully extract the added logic to a HOF. Would help with the isolation / separation of concerns.

Quickly prototyped, but what I have in mind:

```ts
        let internalHandler = async (req, responseToolkit) =>
          await this.handle({
            routeSchemas,
            request: req,
            responseToolkit,
            handler: this.enhanceWithContext(handler),
          });
        
        if(eluTimings) {
          internalHandler = addEluTimings(internalHandler)
        }
        

        this.routes.push({
          handler: internalHandler,
          method,
          path: getRouteFullPath(this.routerPath, route.path),
          options: validOptions(method, route),
        });
```